File: data/python/src/function_sortedcontainers_SortedList_remove.py
from sortedcontainers import SortedList

class HanSeok:
    def remove_element_from_list(self, value: int = 0, input_list: str = None) -> str:
        if input_list is None:
            input_list = "5,3,8,1"
        input_list = input_list.split(",")
        input_list = [int(x) for x in input_list]
        sorted_list = SortedList(input_list)
        try:
            sorted_list.remove(value)
        except ValueError:
            return "Value not found"
        return ",".join(map(str, sorted_list))

File: code/generate_strategies/IR(CoT)/java/java/function_sortedcontainers_SortedList_remove.java
package com.example;

import com.google.common.collect.TreeMultiset;  

public class HanSeok {

    
    public String remove_element_from_list(int value, String input_list) {
        
        if (input_list == null) {
            input_list = "5,3,8,1";
        }
        
        String[] parts = input_list.split(",");
        TreeMultiset<Integer> sortedList = TreeMultiset.create();
        for (String part : parts) {
            sortedList.add(Integer.parseInt(part.trim()));
        }
        
        boolean removed = sortedList.remove(value);
        if (!removed) {
            return "Value not found";
        }
        
        StringBuilder sb = new StringBuilder();
        for (Integer num : sortedList) {
            if (sb.length() > 0) {
                sb.append(",");
            }
            sb.append(num);
        }
        return sb.toString();
    }
}